src/libraries/react-calendar/components/CalendarDrawer.tsx

```diff
@@ -1,69 +1,24 @@
-import React from 'react'
-import { makeStyles, useTheme, Theme } from '@material-ui/core/styles'
-import Drawer from '@material-ui/core/Drawer'
-import Divider from '@material-ui/core/Divider'
-import IconButton from '@material-ui/core/IconButton'
-import ChevronLeftIcon from '@material-ui/icons/ChevronLeft'
-import ChevronRightIcon from '@material-ui/icons/ChevronRight'
 import CalendarSmall from '../engine_components/CalendarSmall'
+import { styled } from "@mui/system"
+import { createTheme } from '@mui/system'
 
-const drawerWidth = 260
-const useStyles = makeStyles((theme: Theme) => ({
-    hide: {
-        display: 'none',
-    },
-    drawer: {
-        width: drawerWidth,
-        flexShrink: 0,
-    },
-    drawerPaper: {
-        width: drawerWidth,
-    },
-    drawerHeader: {
-        display: 'flex',
-        alignItems: 'center',
-        padding: '0 8px',
-        ...theme.mixins.toolbar,
-        justifyContent: 'flex-end',
-    },
-    calendarSmall: {
-        marginTop: theme.spacing(4),
-        marginRight: theme.spacing(1),
-        marginBottom: theme.spacing(4),
-        marginLeft: theme.spacing(1),
-        minHeight: 265,
-        minWidth: 240,
-        background: theme.palette.background.paper,
-    },
-}))
+const theme = createTheme()
 
-function CalendarDrawer(props: any) {
-    const { open, handleDrawerClose } = props
-    const classes = useStyles()
-    const theme = useTheme()
+const CalendarSmallWrapper = styled('div')({
+    marginTop: 75,
+    marginRight: theme.spacing(1),
+    marginBottom: theme.spacing(4),
+    marginLeft: theme.spacing(1),
+    minHeight: 265,
+    minWidth: 240,
+    background: 'white',
+})
 
+function CalendarDrawer() {
     return (
-        <Drawer
```

Comment:
This element should be responsible for opening and hiding of the side bar, but the close/open button is hidden behind the top Toolbar. Since it was impossible to use, I removed this functionality.

Reply:
It's good, if it's not used then yeah, we can safely delete it.